Documentation/operations/upgrade.rst

```diff
@@ -394,6 +397,8 @@ Helm Options
   exposed NodePort services on the first (preferably private) IPv4 and IPv6 address of each device.
 * Helm option ``enableCiliumEndpointSlice`` has been deprecated and will be removed in a future release.
   The option has been replaced by ``ciliumEndpointSlice.enabled``.
+* The Clustermesh's option ``clustermesh.apiserver.kvstoremesh.enabled`` is now set to ``true`` by default.
+  If you want to disable the KVStoreMesh, set ``clustermesh.apiserver.kvstoremesh.enabled=false`` explicitly during the upgrade.
```

Comment:
I'd go straight to the point.

```suggestion
  To disable the KVStoreMesh, set ``clustermesh.apiserver.kvstoremesh.enabled=false`` explicitly during the upgrade.
```

That's just a note for next time though, no need to change :slightly_smiling_face: